src/components/pages/order/OrderPage.js

```diff
@@ -34,25 +40,35 @@ const OrderPageStyled = styled.div`
     background-repeat: repeat;
     background-position:cover;
     .main-container {
-        display: grid;
-        grid-template-columns: repeat(4, 240px);
-        grid-gap: 76.67px;
-        justify-content: center;
+        display:grid;
+        grid-template-columns: 17% 1fr;
         width:1400px;
-        max-width:1400px;
-        height:80vh;
+        max-width: 1400px;
         margin: 0 auto;
-        padding: 60px 0 80px 0;
-        border-bottom-left-radius: ${theme.borderRadius.extraRound};
-        border-bottom-right-radius: ${theme.borderRadius.extraRound};
-        background-color: ${theme.colors.background_white};
-        box-shadow: rgba(0, 0, 0, 0.45) 0px 25px 20px -20px;
-        box-shadow: inset 0px 0px 12px 0px ${theme.colors.greySemiDark};
-        overflow-y: scroll;
-        .grille-item {
-            height:330px;
-            background-color:${theme.colors.white};
-            border-radius: ${theme.borderRadius.extraRound};
+        .sideBar-left {
+            background-color: ${theme.colors.background_dark};
+            height:80vh;
+            width:100%;
+            border-bottom-left-radius: ${theme.borderRadius.extraRound};
+        }
+        .menu-container {
+            display: grid;
+            grid-template-columns: repeat(4, 240px);
```

Comment:
```suggestion
            grid-template-columns: repeat(4, 1fr);
```

si tu sais que toutes tes MenuCard ont la même taille, 1fr peut suffir puisque la taille/ width de la MenuCard est normalement précisée "dans" MenuCard.

Ce n'est pas de la responsabilité de OrderPage que de décider de la taille de MenuCard. 

Mais c'est de la responsabilité de OrderPage que de décider du layout (disposition des éléments à l'intérieur de OrderPage) et donc de la disposition des MenuCards

Reply:
effectivement je n'avais pas vu cette possibilité.